src/styles/cards.scss

```diff
@@ -0,0 +1,91 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: $font-size-medium;
+    line-height: 18px;
+    margin: 0 0 4px;
+    color: $main-accent-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: $font-size-small;
+    line-height: $line-height;
+    margin: 0 0 16px;
+    color: $text-secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: $font-size-small;
+    color: $main-accent-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    font-weight: 400;
+    font-size: $font-size-medium;
+    color: $text-secondary-color;
+    line-height: 18px;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-accent-color;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    width: 168px;
+    height: 40px;
+    border: 1px solid $color1;
```

Comment:
The variable `$color1` is used for both the border and background color of the `&__buy` element. Ensure that `$color1` is defined in your `variables.scss` file and that it represents the intended color for both properties.